src/cpp/configuration/implementations/Address_configuration.cpp

```diff
@@ -48,6 +119,24 @@ Address::Address(
         ip_ = Address::default_ip();
```

Comment:
Add ip_version as argument for this function, and define a default static ipv6 address.

Reply:
:+1: